packages/web/hooks/ui-config/use-incentivize-pool-config.ts

```
@@ -0,0 +1,110 @@
+            params.isPerpetual ?? false,
+            undefined,
+            (tx) => {
+              if (tx.code) reject(tx.rawLog);
+              else resolve();
+            }
+          );
+        } catch (e) {
+          console.error(e);
+          reject(e instanceof Error ? e.message : e);
+        }
+      });
```

Biome (2.5.1)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)